wcfsetup/install/files/lib/util/DateUtil.class.php

```diff
@@ -492,7 +492,7 @@ public static function validateDate($date)
     public static function getFirstDayOfTheWeek()
     {
         if (self::$firstDayOfTheWeek === null) {
-            self::$firstDayOfTheWeek = \intval(WCF::getLanguage()->get('wcf.date.firstDayOfTheWeek'));
+            self::$firstDayOfTheWeek = (int)WCF::getLanguage(->get('wcf.date.firstDayOfTheWeek'));
```

Comment:
Did you perform the replacement using a naïve regular expression or with a proper tool (e.g. Rector)? If this is a naïve regular expression, then there is a great chance that an incorrect replacement introduces bugs, not just obvious syntax errors.

Reply:
Did most of it by hand. Just missed that one.

Reply:
Before you spend more (manual) time on this: I'm not really sold on this being an overall improvement. As an example the lack of parentheses makes precedence much less clear and `\intval()` is not really in super hot paths anyway. When I last checked with XDebug's profiler, `intval` was not anywhere near the expensive locations.